corehq/apps/enterprise/views.py

```diff
@@ -121,10 +127,12 @@ def security_center(request, domain):
     )
 
     context.update({
-        'reports': [EnterpriseReport.create(slug, request.account.id, request.couch_user) for slug in (
-            EnterpriseReport.API_USAGE,
-            EnterpriseReport.TWO_FACTOR_AUTH,
-        )],
+        'groups': [
+            {'name': '',
```

Comment:
For the sake of consistency, I would prefer that we use a name for this section as well. If we instead want to leave section names are optional, then I think we should modify the markup to only insert the `<h3>` tag when a name is present. Currently, it looks like an empty `<h3>` tag will still be created for the empty name.

Reply:
I cannot think of a good group name for this two tiles. Also, if no name, there won't be an empty `<h3>` tag.

Reply:
Maybe we're talking about different things? From the template:
```
{% for group in groups %}
      <h3>{{ group.name }}</h3>
     ...
{% endfor %}
```

If the group name is empty, we insert an empty `<h3>` element (I also checked the code out to verify this).

I think something simple like "Overview" would work, but if you can't think of a name here, I think we should make that <h3> conditional

Reply:
I see! I find the empty <h3> element. Conditionally display in 0b843a3118e8374676454e6f8a80a3141980882e